osm2lanes-web/src/map.rs

```diff
@@ -37,26 +38,32 @@ impl MapComponent {
     const MAP_ID: &'static str = "map";
 
     fn render_map(&self) -> Html {
-        let container = self.container.clone();
-        if self.is_searching {
-            container.set_class_name("searching");
-        } else {
-            container.set_class_name("");
-        }
-        Html::VRef(container.into())
+        // creating the container here doesn't work
+        // modifying the container here breaks things
+        // regardless, it is unclear if this clone is OK
+        Html::VRef(self.container.clone().into())
     }
 }
 
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct MapOptions {
+    scroll_wheel_zoom: bool,
+}
+const MAP_OPTIONS: MapOptions = MapOptions {
+    scroll_wheel_zoom: false,
```

Comment:
Aw, I kind of miss this. Double clicking to zoom feels a bit too similar to clicking to focus somewhere. But I get the tension between scrolling the page and map

Reply:
yeah, I wanted to try and get ctrl+scroll working, but I didn't see an option for that in leaflet

Reply:
https://github.com/elmarquis/Leaflet.GestureHandling

that's what I want!
I couldn't find it because it is not part of leaflet natively...